bot_file.py

```diff
@@ -404,6 +406,34 @@ def elm_progress_on(self, channel: str, branch_name: str) -> None:
 
         self.send_channel_message(channel, message)
 
+    def find_elm_017_matches(self, channel: str, filename_pattern: str) -> None:
+        """ give a filename of elm to get me to tell you how it looks on master """
+
+        self.repo.get_ready()
+        message = "We have found the following filenames:\n"
+
+        filenames = self.repo.get_files_for_017(filename_pattern)
```

Comment:
In the `TodayIDidBot`, everything is targetted at NRI - I spent a while last night untangling the logic, so now there's a core `GenericBot`. So I'd prefer to keep this as `self.repo.get_elm_filenames_for_017` than to introduce mixins right now (want mixins, but there's a pattern for extending a class via registration to certain classes I want to try out first)